assets/css/styles.scss

```diff
@@ -141,6 +140,9 @@ body {
 
 p {
   margin: 0 0 1em 0;
+  .img {
+    text-align: center;
```

Comment:
Setting `text-align: center` on an `img` won't work since that property only applies to the *contents* of an element. Suggest you set `display: block` (to permit setting the margin) and `margin: 0 auto 0 auto` (which sets the left and right margins be nice and spacious)